nifi-system-tests/nifi-system-test-suite/src/test/java/org/apache/nifi/tests/system/SpawnedStandaloneNiFiInstanceFactory.java

```diff
@@ -102,7 +103,7 @@ public void start(final boolean waitForCompletion) {
                 if (waitForCompletion) {
                     waitForStartup();
                 }
-            } catch (IOException e) {
+            } catch (IOException | GeneralSecurityException e) {
```

Comment:
This is another reason that it would be helpful to catch and wrap the `GeneralSecurityException`, so that this change can be avoided.